### ISSUE TITLE
Missing captions on last two figures of 10-3

### ISSUE THREAD

**Issue by marcozullich** [Atcold/pytorch-Deep-Learning#493]
On `10-3.md` the last two figures are missing captions:
* would-be Fig. 6 (in section "Detailed Model Structure") has the caption superimposed on the image file, but this causes issues in the translated versions since there's only an English sentence which might not be intelligible for the foreign readers
* would-be Fig. 7 (in section "Examples of Movement"), which is a composition of 4 sub-images, has no caption at all: although the caption is inferrable from the text above, this doesn't seem to fit with the style of the project: as far as I've seen, all figures are numbered and have a caption, even when this is trivially obtainable from the text.

**Comment by Atcold** [comment]
What do you suggest?

**Comment by marcozullich** [comment]
I could propose a caption for Fig. 7 and update the EN document accordingly, then write in all of the slack groups for regional translations asking to propagate the update.

**Comment by Atcold** [comment]
Sounds good. Please, proceed.

**Comment by marcozullich** [comment]
See #661

**Comment by marcozullich** [comment]
Memo:
Asked for corrections in

- [ ] ES
- [ ] FR
- [ ] JA
- [ ] KO
- [ ] TR
- [ ] ZH

**Comment by mvcaro** [comment]
I noticed 10-3 had two different Fig. 5, each under Emulator and Controller sections; the following changes have been made (#743):

1. 'Fig. 5' under Controller section has now become 'Fig. 6'
2. Under Controller section 'Refer to Figure 5' has been changed to 'Refer to Figure 6'
3. 'Fig. 6' under Detailed Model Structure section has been updated to 'Fig. 7' and captions were added
4. As the figure under Detailed Model Structure section had a caption within the image, the caption within the image has been removed to avoid confusion (pytorch-Deep-Learning-master/docs/images/week10/10-3/fig14.png)
5. 'Fig. 7' under Detailed Model Structure section has been updated to 'Fig. 8' and captions were added

@marcozullich if you agree with these changes, then 1, 2, 3 and 4 need to be updated across all languages except [EN] and [ES]

**Comment by marcozullich** [comment]
Yep, I'm going for it for IT, will request changes for all other languages. Thanks! @lbourdois @JonathanSum @seungjaeryanlee @i-am-mel-dev would you be so kind as to update the figure numbering as per @mvcaro's comment?

**Comment by lbourdois** [comment]
Thank you for the information,
I will add the change in my next PR.

De : Marco Zullich <notifications@github.com>
Envoyé : mardi 5 janvier 2021 08:41
À : Atcold/pytorch-Deep-Learning <pytorch-Deep-Learning@noreply.github.com>
Cc : lbourdois <loick.bourdois@hotmail.com>; Mention <mention@noreply.github.com>
Objet : Re: [Atcold/pytorch-Deep-Learning] Missing captions on last two figures of 10-3 (#493)


Yep, I'm going for it for IT, will request changes for all other languages. Thanks! @lbourdois<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Flbourdois&data=04%7C01%7C%7Cc4cb6c7792f3459d1c5d08d8b14d4b78%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637454292823827485%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ml5f4xB%2BBc4ktsl5h0an8gOWPFxe1qDj06Csdzx9PJ4%3D&reserved=0> @JonathanSum<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FJonathanSum&data=04%7C01%7C%7Cc4cb6c7792f3459d1c5d08d8b14d4b78%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637454292823827485%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=8NPt7nGlsRPzNtczkBmJQWd6h3lwbdI3RLGRZ5R207E%3D&reserved=0> @seungjaeryanlee<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fseungjaeryanlee&data=04%7C01%7C%7Cc4cb6c7792f3459d1c5d08d8b14d4b78%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637454292823837454%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=y%2F9l%2BHzYEAhfvufQV98%2FVtaI0Q52AyaFgnCUuyThzHo%3D&reserved=0> @i-am-mel-dev<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fi-am-mel-dev&data=04%7C01%7C%7Cc4cb6c7792f3459d1c5d08d8b14d4b78%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637454292823837454%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=NohjP573jpY4nFCrvIHmUGtdJjjl1iizseI0GTmPhOY%3D&reserved=0> would you be so kind as to update the figure numbering as per @mvcaro<https://emea01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmvcaro&data=04%7C01%7C%7Cc4cb6c7792f3459d1c5d08d8b14d4b78%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637454292823837454%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=uIei%2FYJIdfYaK4aw419gQEbDZ5sUd7sgCsIdOiOmXfQ%3D&reserved=0>'s comment?

-

**Comment by JonathanSum** [comment]
Updated #733
